cmd/tle/commands/encrypt.go

```diff
@@ -42,164 +42,81 @@ func Encrypt(flags Flags, dst io.Writer, src io.Reader, network *http.Network) e
 		return tlock.Encrypt(dst, src, flags.Round)
 
 	case flags.Duration != "":
-		durations, err := parseDurations(flags.Duration)
+		start := time.Now()
+		totalDuration, err := parseDurationsAsSeconds(start, flags.Duration)
 		if err != nil {
 			return err
 		}
-		now := time.Now()
-		decryptionTime := durations.from(now)
-		if decryptionTime.Before(now) || decryptionTime.Equal(now) {
+
+		decryptionTime := start.Add(time.Duration(totalDuration) * time.Second)
```

Comment:
Notice that `time.Duration` has a [.Seconds() method](https://pkg.go.dev/time#Duration.Seconds) for whenever you really need the number of seconds, why not just return a time.Duration above? (see my other comment too)